app/common/routes/index.js

```diff
@@ -79,10 +89,20 @@ export const configureRoutes = ({ store }) => { // eslint-disable-line
               />
             </Route>
             <Route path="sign-in" component={SignInPage} />
+            <Route path="update-factor" component={UpdateFactorSignInPage} />
+
             <Route path="reset" component={ResetPasswordPage} />
             <Route path="reset/:id" component={NewPasswordPage} />
             <Route onEnter={requireAuth}>
               <Route path="accept" component={AcceptPage} />
+              <Route path="otp-send" component={OtpPage} />
+              <Route path="request-factor" component={RequestFactorPage} />
+              <Route path="request-factor/approve" component={RequestFactorOtpPage} />
```

Comment:
Mb will be better `RequestFactorApprovePage` ?

Reply:
+